src/index.ts

```diff
@@ -308,6 +308,11 @@ export const TOO_MANY_REQUESTS = 429;
  *
  * */
 export const UNAUTHORIZED = 401;
+/**
+ * @deprecated Please use StatusCodes.UNAVAILABLE_FOR_LEGAL_REASONS
+ *
+ * */
+export const UNAVAILABLE_FOR_LEGAL_REASONS = 451;
```

Comment:
Although it doesn't make sense that this is deprecated, I needed to add this or the tests would fail:

```
  ● Backwards compatability with v1 › Gets codes exported by constant

    expect(received).toBe(expected) // Object.is equality

    Expected: 451
    Received: undefined

      10 |     codes.forEach((o) => {
      11 |       expect((HttpStatusV1 as any)[o.constant]).toBe(o.code);
    > 12 |       expect((HttpStatusV2 as any)[o.constant]).toBe(o.code);
         |                                                 ^
      13 |     });
      14 |   });
      15 | 

      at src/index.test.ts:12:49
          at Array.forEach (<anonymous>)
      at Object.<anonymous> (src/index.test.ts:10:11)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 8 passed, 9 total
Snapshots:   0 total
Time:        0.725 s, estimated 1 s
Ran all test suites matching /.\/src/i.
error Command failed with exit code 1.

Reply:
Ah yeah I think I know the problem. These tests iterate over all codes to ensure backwards compatibility, but that will be increasingly untrue as we continue to add new codes. I can change the tests to fix this.